azurerm/internal/services/containers/kubernetes_cluster_resource.go

```diff
@@ -500,6 +500,14 @@ func resourceKubernetesCluster() *schema.Resource {
 										Optional: true,
 									},
 
+									"azure_rbac_enabled": {
```

Comment:
should this be
```suggestion
									"azure_ad_rbac_enabled": {
```

to make it obvious its AzureAD not some other form of azure RABC?

Reply:
I don't think so, as it actually is/enables Azure RBAC to be used for Kubernetes. What it means is that you can give a user a role in the portal (i.e. `Azure Kubernetes Service RBAC Viewer`) with scope (i.e. `$AKS_ID/namespaces/<namespace-name>`). 

The connection with Azure AD is that the cluster should have managed Azure AD integration enabled to facilitate this, but those mechanics are from a user point of view abstracted to Azure RBAC.

Reply:
👍